addons/binding/org.openhab.binding.roku/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,28 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Roku Binding
+Bundle-SymbolicName: org.openhab.binding.roku;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.1.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-ClassPath: .
+Import-Package: 
+ org.apache.commons.collections,
+ org.apache.commons.io,
+ org.eclipse.smarthome.config.core,
+ org.eclipse.smarthome.config.discovery,
+ org.eclipse.smarthome.core.library.types,
+ org.eclipse.smarthome.core.thing,
+ org.eclipse.smarthome.core.thing.binding,
+ org.eclipse.smarthome.core.thing.binding.builder,
+ org.eclipse.smarthome.core.thing.type,
+ org.eclipse.smarthome.core.types,
+ org.jupnp.model.meta,
+ org.openhab.binding.roku,
+ org.openhab.binding.roku.internal.handler,
+ org.slf4j
+Service-Component: OSGI-INF/*.xml
+Export-Package: org.openhab.binding.roku,
+ org.openhab.binding.roku.internal.handler
```

Comment:
You should not export your internal's, either they should not be internal or not exported